src/main/java/com/jcabi/aether/RootArtifact.java

```diff
@@ -121,6 +129,47 @@ public Artifact artifact() {
     }
 
     /**
+     * Downloads parent artifact from repositories.
+     * @param system The Repository System
+     * @param session The Repository Session
+     * @param repos Repositories to send request
+     * @return Parent artifact
+     * @throws Exception if downloading fails
+     */
+    public Artifact lookUpParent(final RepositorySystem system,
+        final RepositorySystemSession session,
+        final List<RemoteRepository> repos)
+        throws Exception {
+        Artifact result = null;
```

Comment:
@piddubnyi can we avoid `null` here? Please read http://www.yegor256.com/2014/05/13/why-null-is-bad.html